features/step_definitions/user_steps.rb

```diff
@@ -17,6 +17,19 @@
   }
 end
 
+Given(/^I there register as a user with username "([^"]*)" and email "([^"]*)"$/) do |name, email|
+  steps %Q{
+    When I fill in:
+      | element                    | content          |
+      | Name                       | #{name}           |
+      | Email                      | #{email} |
+      | Address                    | Fjällgatan 3, Göteborg, Västa Götaland, Sverige |
+      | Password                   | password         |
+      | Password confirmation      | password         |
+    And I click the "Register" button
+  }
+end
```

Comment:
I would prefer you guys used the Device/Warden helpers here rather than going through the interface. Not a blocker though.